packages/web/docs/src/pages/self-hosting/get-started.mdx

```diff
@@ -89,23 +95,151 @@ the environment variables.
 
 </Callout>
 
-After setting the environment variables, you can start the GraphQL Hive services using
-`docker-compose`.
+After setting the environment variables, pull the required images for GraphQL Hive services. This's
+gonna take some time if you're doing it for the first time.
+
+```bash
+docker compose -f docker-compose.community.yml pull
+```
+
+After it's done, you can start the GraphQL Hive services using `docker compose`.
 
 ```bash
-docker-compose -f docker-compose.community.yml up
+docker compose -f docker-compose.community.yml up
 # or of you embedded the environment variables
-docker-compose -f docker-compose.with-env.yml up
+docker compose -f docker-compose.with-env.yml up
 ```
 
 Wait until all the services are up and running.
 
 Congratulations 🥳, you just started your own GraphQL Hive instance.
 
+You'll notice the folder named `.hive` at your root directory that has been mounted from docker
+representing the volumes for the different storages used by Hive services like `postgres` and
```

Comment:
Please mention here that this directory is acting as a volume for storing the data from Hive dependencies: database and storage. Also mention that deleting this directly will end up with a loss of data.